Kitodo/src/main/resources/hibernate.cfg.xml

```diff
@@ -24,13 +24,18 @@
         <property name="hibernate.connection.driver_class">com.mysql.jdbc.Driver</property>
 
         <property name="hibernate.connection.url">
-            jdbc:mysql://localhost/kitodo
+            jdbc:mysql://localhost/kitodo?useSSL=false
         </property>
         <property name="hibernate.connection.username">kitodo</property>
         <property name="hibernate.connection.password">kitodo</property>
         <property name="hibernate.connection.autoReconnect">true</property>
         <property name="hibernate.connection.autoReconnectForPools">true</property>
 
+        <!-- SSL connection properties -->
+        <property name="hibernate.connection.requireSSL">false</property>
+        <property name="hibernate.connection.verifyServerCertificate">false</property>
+        <property name="hibernate.connection.useSSL">false</property>
```

Comment:
Why this property is defined here and also in the URL?

Reply:
It must be defined on both as it used in different ways inside Hibernate and it is depending on used MySQL Server version.

Reply:
Explanation: the value of the property is used on other places where the URL is not interpreted and the value in the URL is not set as property which is used on other places. So it must be defined on two places.